MANIFEST

```diff
@@ -0,0 +1,32 @@
+AUTHORS
+bin/wg-meta
+CHANGES
+COPYRIGHT
+lib/Wireguard/WGmeta.pod
```

Comment:
hmmm having a pod without matching library seems odd ... I would probaly add this one directly into the wg-meta.pl file